requirements.txt

```diff
@@ -182,28 +200,48 @@ protobuf==3.20.3; sys_platform == "linux" and python_version > "3.8"
 psutil==5.9.4; sys_platform == "linux" and python_version > "3.8" 
 ptyprocess==0.7.0; sys_platform == "linux" and python_version > "3.8" 
 pure-eval==0.2.2; sys_platform == "linux" and python_version > "3.8" 
+pycollada==0.7.2; sys_platform == "linux" and python_version > "3.8" 
 pyflakes==3.0.1; sys_platform == "linux" and python_version > "3.8" 
+pyglet==1.5.27; sys_platform == "linux" and python_version > "3.8" 
 pygments==2.14.0; sys_platform == "linux" and python_version > "3.8" 
 pymcubes==0.1.4; sys_platform == "linux" and python_version > "3.8" 
 pyparsing==3.0.9; sys_platform == "linux" and python_version > "3.8" 
 pyproject-autoflake==1.0.2; sys_platform == "linux" and python_version > "3.8" 
 python-dateutil==2.8.2; sys_platform == "linux" and python_version > "3.8" 
+python-fcl==0.7.0.5; sys_platform == "linux" and python_version > "3.8" 
 pytz==2022.7.1; sys_platform == "linux" and python_version > "3.8" 
+pyyaml==6.0.1; sys_platform == "linux" and python_version > "3.8" 
+referencing==0.30.2; sys_platform == "linux" and python_version > "3.8" 
 requests==2.28.2; sys_platform == "linux" and python_version > "3.8" 
+rich==13.6.0; sys_platform == "linux" and python_version > "3.8" 
+rpds-py==0.10.6; sys_platform == "linux" and python_version > "3.8" 
+rtree==1.0.1; sys_platform == "linux" and python_version > "3.8" 
+scikit-image==0.22.0; sys_platform == "linux" and python_version > "3.8" 
 scikit-learn==1.2.2; sys_platform == "linux" and python_version > "3.8" 
 scipy==1.10.1; sys_platform == "linux" and python_version > "3.8" 
+shapely==2.0.2; sys_platform == "linux" and python_version > "3.8" 
+signature-dispatch==1.0.1; sys_platform == "linux" and python_version > "3.8" 
 six==1.16.0; sys_platform == "linux" and python_version > "3.8" 
 soupsieve==2.4; sys_platform == "linux" and python_version > "3.8" 
 stable-baselines3==1.8.0; sys_platform == "linux" and python_version > "3.8" 
 stack-data==0.6.2; sys_platform == "linux" and python_version > "3.8" 
+svg-path==6.3; sys_platform == "linux" and python_version > "3.8" 
+sympy==1.12; sys_platform == "linux" and python_version > "3.8" 
 threadpoolctl==3.1.0; sys_platform == "linux" and python_version > "3.8" 
+tifffile==2023.9.26; sys_platform == "linux" and python_version > "3.8" 
 timeout-decorator==0.5.0; sys_platform == "linux" and python_version > "3.8" 
 tokenize-rt==5.0.0; sys_platform == "linux" and python_version > "3.8" 
 toml==0.10.2; sys_platform == "linux" and python_version > "3.8" 
 tomli==2.0.1; sys_platform == "linux" and python_version > "3.8" 
 torch==1.12.1; sys_platform == "linux" and python_version > "3.8" 
 traitlets==5.9.0; sys_platform == "linux" and python_version > "3.8" 
+trimesh[easy,recommend]==4.0.0; sys_platform == "linux" and python_version > "3.8" 
```

Comment:
It did not add this for the other platforms. Can I do anything to add it to all of them?

Reply:
This is still a problem. I additionally added the dependency to `pyproject.toml`.